lib/notifications/display.dart

```diff
@@ -354,10 +355,9 @@ class NotificationDisplayManager {
 
     assert(debugLog('  account: $account, narrow: $narrow'));
     // TODO(nav): Better interact with existing nav stack on notif open
-    navigator.push(MaterialAccountWidgetRoute<void>(accountId: account.id,
+    unawaited(navigator.push(MaterialAccountWidgetRoute<void>(accountId: account.id,
```

Comment:
nit in commit message: duplicate text

    The `Future` that `Navigator.push` returns resolves when a
    corresponding `Navigator.pop` is called.
    
    The future Navgiator.push returns completes when the there is a
    corresponding Navigator.pop call.  We don't use that as of now, so we
    use unawaited to explicitly discard it.